rfcs/rfc-filing-management.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: 2020-08-30
+- Target Major Version: (EPIC or User Story TAG|Link)
+- Reference Issues: (fill in existing related issues, if any)
+- Entity Issue: (leave this empty)
+- Implementation PR: (leave this empty)
+- Submitters: Lekshmi Mallika and Thor Wolpert
+
+# Summary
+
+This expands on [rfc-legal-registry-ledger](rfc-legal-registry-ledger.md) to handle filings specifically.
+
+The approach will start to follow a somewhat hexagonal architecture, with the current model abstracted and include the filing document as core.
+
+This approach will both maintain the submission filing as part of the audit history and produce an equivalent filing in the most recent JSONSchema version. This will allow conversion to pdf and other such translation to support a much more limited set versions, which is desireable.
```

Comment:
Did we want to generate (on-the-fly) historical PDFs (eg, the PDF as it looked in the past)? How would this be handled?
(As compared with, say, generating PDFs during filing completion and storing them for future retrieval.)